subsys/bluetooth/Kconfig.iso

```
@@ -99,6 +99,8 @@ config BT_ISO_TX_MTU
 	range 1 4095
 	help
 	  Maximum MTU for Isochronous channels TX buffers.
+	  This is the actual data payload. It doesn't include the optional
+	  HCI ISO Data packet fields (ie. `struct bt_hci_iso_ts_data_hdr`)
```

```suggestion
	  HCI ISO Data packet fields (e.g. `struct bt_hci_iso_ts_data_hdr`)
```

The header can also be `struct bt_hci_iso_data_hdr` depending on the use, thus `e.g.` instead of `i.e.`

---

Thank you, I fixed BT_ISO_TX_MTU description